src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -1448,6 +1449,45 @@ void ov::npuw::util::gather(const ov::SoPtr<ov::ITensor>& src,
     }
 }
 
+ov::SoPtr<ov::ITensor> ov::npuw::util::view(const ov::SoPtr<ov::ITensor>& src,
+                                            const ov::npuw::util::View& from,
+                                            const ov::npuw::util::View& to) {
+    const auto type = src->get_element_type();
+    NPUW_ASSERT(from.size() == to.size());
+
+    // Sub-byte views are not supported here
+    NPUW_ASSERT(type != ov::element::u4 && type != ov::element::i4);
+
+    const auto num_dims = from.size();
+    ov::Shape view_shape;
+    for (auto d = 0u; d < num_dims; d++) {
+        view_shape.push_back(to[d] - from[d]);
+    }
+
+    const auto strides = src->get_strides();
```

Comment:
I'm not sure if remote tensors support strides (as far as I know L0 tensors don't). So it should be ok for weightless subgraphs. However with adding ins/outs/intermediate tensors to remote memory, wouldn't it break?

Reply:
> I'm not sure if remote tensors support strides (as far as I know L0 tensors don't)

It works perfectly there - tested. Actually, L0 tensors are plain buffers and strides are just logical distances here in a big plain buffer. There's no reason for this not to work.